db/seeds.rb

```diff
@@ -11,11 +11,13 @@ def create_users()
     User.create( 
       first_name:   Faker::Name.first_name, 
       last_name:    Faker::Name.last_name,
-      linkedin_id:  Faker::Lorem.characters(10),
+      description:  Faker::Lorem.paragraph,
       contact_url:  Faker::Internet.url,
-      visible:      Faker::Boolean.boolean(0.8),
-      role:         :user,
-      description:  Faker::Lorem.paragraph
+      provider:     :linkedin,
+      linkedin_id:  Faker::Lorem.characters(10),
```

Comment:
We're going to remove this field eventually so not sure if we want to keep it in this file? Doesn't really matter at the moment

Reply:
I wasn't able to create a record without including it

Reply:
Once you merge, that won't be a problem anymore as the changes from the recently merged authentication branch took away the presence validation on this linked_in attribute.